### PR TITLE
Fix py files for ConfDB parsing (90X)

### PR DESCRIPTION
**Pull request by Martin-Grunewald**
Fix py file for ConfDB parsing (90X)

### CODE REVIEW

**Comment by Martin-Grunewald** [comment]
please test

**Comment by Martin-Grunewald** [comment]
please test

**Comment by Martin-Grunewald** [comment]
type bug-fix

**Comment by mmusich** [comment]
+1

File: Alignment/LaserAlignment/python/HLTLaserAlignmentEventFilter_cfi.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import FWCore.ParameterSet.Config as cms
+
+
+from Alignment.LaserAlignment.LaserAlignmentDefs_cff import *
```
Comment (davidlange6) [inline Alignment/LaserAlignment/python/HLTLaserAlignmentEventFilter_cfi.py]: why is this needed? (we don't include cffs into cfis...)

Comment (Martin-Grunewald) [inline Alignment/LaserAlignment/python/HLTLaserAlignmentEventFilter_cfi.py]: This cff factors out a bunch of constants reused also elsewhere.

Comment (davidlange6) [inline Alignment/LaserAlignment/python/HLTLaserAlignmentEventFilter_cfi.py]: ok, but then the users should call the cff. cfis there there to define a module (and this module doesn't appear to need the constants)

**Comment by Martin-Grunewald** [comment]
please test
